src/llmcompressor/arg_parser/README.md

```diff
@@ -0,0 +1,45 @@
+# Argument parser for `oneshot`, `train`, `eval` entrypoints
+
+Argument parsers in `llm-compressor` define the input arguments required for various entrypoints, including `oneshot`, `train`, and `eval`.
```

Comment:
Why are we calling these argument parsers?

Reply:
Bc they are argument parsers no?